gnetlist/src/s_traverse.c

```diff
@@ -357,7 +357,7 @@ s_traverse_net (NET *nets, int starting, OBJECT *object, char *hierarchy_tag, in
 
   if (object->type == OBJ_PIN) {
 
-    verbose_print (starting ? "p" : "P");
+    starting ? verbose_print ("p") : verbose_print ("P");
```

Comment:
Namely `error: invalid conversion from ‘const char*’ to ‘char*’`. An error, not a warning as in other string conversion cases.